src/Tools/BuildActionTelemetryTable/Program.cs

```diff
@@ -498,6 +498,7 @@ internal static ImmutableArray<TelemetryInfo> GetTelemetryInfos(ImmutableArray<T
             return codeActionAndProviderTypes
                 .Distinct(FullNameTypeComparer.Instance)
                 .Select(GetTelemetryInfo)
+                .OrderBy(info => info.Item1)
```

Comment:
This is the only functional change which sorts based on TypeName.

Reply:
nit, consdier named tuple fields to make this clearer.